### ISSUE TITLE
ARIA menu error on community and item pages

### ISSUE THREAD

**Issue by lgeggleston** [DSpace/dspace-angular#5488]
## Describe the bug
Accessibility issue appearing when running community and item pages through the [WAVE tool](http://wave.webaim.org/):
- On community and item page: broken ARIA menu error 
    - this looked as though it's caused by the edit options menubar item still existing in the HTML, just hidden when the user doesn't have permissions.  If that's the case, I think it may need additional ARIA labeling to mark it to ignore for screen readers etc.

Observed on Sandbox 10.0, relevant to Testathon T051. Also observed on Demo (9.x)

## To Reproduce
1. Get the WAVE browser extension
2. Run it (on Sandbox) for a community page and an item page

## Expected behavior
Likely additional ARIA labeling to make the hidden menu ignorable by screen readers
No issues found, or existing issues known and accepted.  

## Related work
Link to any related tickets or PRs here.

**Comment by guillermo-escire** [comment]
Hi @lgeggleston , I'm a partner at @jtimal, could you please assign me this issue to work on?

**Comment by lgeggleston** [comment]
Thanks for volunteering @guillermo-escire! You're assigned.
